Gemfile.lock

```diff
@@ -0,0 +1,139 @@
+GEM
```

Comment:
I just learned that the `Gemfile.lock` shouldn't be checked into the version control because it would enforce apps which uses the gem to have the exact same versions. Source: https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/ It's a quite outdated post but I guess things haven't changed.

Reply:
Okay, I'm removing the file from this PR and adding a reference on a future PR adding the .gitignore file